### PR TITLE
Added more tests to services/images.go

### PR DESCRIPTION
**Pull request by omaciel**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Review by acosferreira — APPROVED** [review]
lgtm

File: pkg/services/images.go [hunk 1]
```diff
@@ -872,11 +872,16 @@ func (s *ImageService) UpdateImageStatus(image *models.Image) (*models.Image, er
...
+	// Search for an organization with an image with specific name
 	result := db.Org(orgID, "").Where("(name = ?)", name).First(&imageFindByName)
+	// If we get an error from the query
 	if result.Error != nil {
+		// If no records were found
 		if result.Error == gorm.ErrRecordNotFound {
+			// Return false but don't return the error
 			return false, nil
 		}
+		// If a different type of eeror happens, return it
```
Comment (de1987) [inline pkg/services/images.go:884]: typo
```suggestion
		// If a different type of error happens, return it
```

**Review by de1987 — DISMISSED** [review]

File: pkg/services/images.go (hunk 1 above)
Comment (omaciel) [reply pkg/services/images.go]: Fixed

File: pkg/services/images_test.go [hunk 2]
```diff
@@ -176,6 +176,43 @@ var _ = Describe("Image Service Test", func() {
...
+				image := models.Image{}
+				error := service.CreateImage(&image)
+				Expect(error).To(MatchError(new(services.OrgIDNotSet)))
+			})
+			It("should raise ImageNameUndefined", func() {
+				image := models.Image{OrgID: faker.UUIDHyphenated()}
+				error := service.CreateImage(&image)
+				Expect(error).To(MatchError(new(services.ImageNameUndefined)))
+			})
+			It("should raise ImageNameAlreadyExists", func() {
```
Comment (mgold1234) [inline pkg/services/images_test.go:191]: this use case test here -https://github.com/mgold1234/edge-api/blob/main/pkg/routes/images_test.go#L254
is it good enough?

File: pkg/services/images_test.go [hunk 3]
```diff
@@ -176,6 +176,43 @@ var _ = Describe("Image Service Test", func() {
 			})
 		})
 	})
+	Describe("#CreateImage", func() {
+		Context("when creating a new image", func() {
+			It("should raise OrgIDNotSet", func() {
```
Comment (mgold1234) [inline pkg/services/images_test.go:181]: we check it here -https://github.com/mgold1234/edge-api/blob/main/pkg/services/images_test.go#L488

Comment (omaciel) [reply pkg/services/images_test.go:181]: The example you linked is a mocked out response where's this test specifically calls out `CreateImage` and allows the function to naturally handle the problem, returning the expected error. Does that make sense?

File: pkg/services/images_test.go (hunk 2 above)
Comment (omaciel) [reply pkg/services/images_test.go:191]: Same as before, the example you linked is a mocked out response where's this test specifically calls out CreateImage and allows the function to naturally handle the problem, returning the expected error. Does that make sense?

Comment (mgold1234) [reply pkg/services/images_test.go:191]: sure, thanks

**Review by mgold1234 — APPROVED** [review]

**Pull request merged**